### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> See less button]: Visual name is different from accessible name for 'See More' button.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7785]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the 'See more' button.
4. Observe the issue.

**Actual Result:**
For 'See more' button, visual name is different from the accessible name. Screen reader is announcing as **'More app bar button'.**

**Expected Result:**
For 'See more' button, visual name should be similar to the accessible name. Screen reader should announce as **''See more Button".**

**Observation:**
Same issue repro for 'See less Button' when user once activates the 'See more Button'.

**User Impact:**
Person with cognitive disability will get impacted if visual name is different from accessible name due to which user will have difficulty in tracking the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.05.03

**“Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

<img width="960" alt="Code Snippet ~ Visual name id different from the accessible name for see more button" src="https://user-images.githubusercontent.com/104483005/187355324-920e9645-85b9-4374-9566-ff1cdd913ba3.png">

https://user-images.githubusercontent.com/104483005/187356097-f5e0b61a-ce55-4672-b4ff-4c92bde423af.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

This is a bug on a WinUI 2 control. For this button, we are utilizing `CommandBar.SecondaryCommands` and the button is not generated within our codebase.

```
<CommandBar.SecondaryCommands>
   <AppBarToggleButton
        Label="Use fixed dimensions"
        IsChecked="{Binding UseFixedDimensions, Mode=TwoWay}"/>
</CommandBar.SecondaryCommands>
```

The behavior of our app is also consistent with the behavior in the WinUI 2 Gallery.

![CommandBar](https://user-images.githubusercontent.com/98650930/206555647-0cc270e4-5342-4190-8204-f0e6d47e73fd.png)

Please let me know if you have any questions!

**Comment by vagpt** [comment]
Hi @anna-dingler,

Can you please let us know the point of contact for the WinUI 2 control team as this bug is external to them.

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

I'm still working on getting a WinUI 2 contact.

In the meantime, I'm going to close this issue because we do not release the UWP visualizer as an app for customers to use. We use this app for internal testing. Only accessibility issues found in the preview window will reach our customers because these are caused by the Adaptive Cards UWP rendering package.

Please let me know if you have any questions - thanks!

**Comment by vaishali1397** [comment]
Closing this issue and adding A11yTTValidated tag as per comment.
![image](https://user-images.githubusercontent.com/104483005/235416769-6e687394-8f1d-4e3e-8c2e-e0ae3a437b02.png)
